mtp_api/apps/core/management/commands/dump_and_upload_for_ap.py

```diff
@@ -0,0 +1,53 @@
+import textwrap
+import datetime
+from tempfile import NamedTemporaryFile
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Dump and upload data to S3 for Analytical Platform
+    """
+    help = textwrap.dedent(__doc__).strip()
+
+    def add_arguments(self, parser):
+        parser.add_argument('bucket_name', help='AWS S3 global bucket name')
+
+    def handle(self, *args, **options):
+        file_path_credits = credits_temp_file_generation()
+        file_path_disbursements = disbursements_temp_file_generation()
+
+        bucket_name = options['bucket_name']
+
+        object_name = todays_date()
```

Comment:
`object_name` is perhaps not the best name for this. it makes the `f'{object_name}_credits'` further down not really do what it's saying. maybe `date` or `date_range`?